addons/im_livechat/static/src/embed/common/chatbot/chatbot_service.js

```diff
@@ -38,6 +41,13 @@ export class ChatBotService {
                 this.start();
             }
         });
+        this.livechatInitializedService.ready.then(() => {
+            this.busService.subscribe("im_livechat.bot_command", (payload) => {
+                this.store.insert(payload);
+                this.chatbot.restart();
+                this._triggerNextStep();
+            });
+        });
```

Comment:
I'd rather have a `mail.record/insert` and an `onChange` callback on the `operator` field to start the chatbot. It's better to avoid highly specific code and get closer to the discuss code. This approach will also allow us to remove the imperative calls to the `start` method above.

Reply:
I don't think it is reliable. `operator` doesn't get updated if the same bot was initiated twice in a row. And `chatbot` gets updated quite often. It's really tough to determine which update was caused by the `/bot` command.